website/content/docs/k8s/installation/vault/data-integration/server-tls.mdx

```diff
@@ -51,139 +51,129 @@ which also uses an intermediate signing authority.
-
-```shell-session
-$ vault write pki/roles/consul-server \
-    allowed_domains="<Allowed-domains-string>" \
-    allow_subdomains=true \
-    allow_bare_domains=true \
-    allow_localhost=true \
-    generate_lease=true \
-    max_ttl="720h"
-```
-
-To generate the `<Allowed-domains-string>` use the following script as a template:
-
-```shell-session
-#!/bin/sh
-
-# NAME is set to either the value from `global.name` from your Consul K8s value file, or your $HELM_RELEASE_NAME-consul
-export NAME=consulk8s
-# NAMESPACE is where the Consul on Kubernetes is installed
-export NAMESPACE=consul
-# DATACENTER is the value of `global.datacenter` from your Helm values config file
-export DATACENTER=dc1
-
-echo allowed_domains=\"$DATACENTER.consul, $NAME-server, $NAME-server.$NAMESPACE, $NAME-server.$NAMESPACE.svc\"
-```
-
-### Link the Vault policies to Consul workloads
-Create three Vault auth roles, one for the Consul servers, one for the Consul clients, and one for Consul components, that link the policy to each Consul workload on Kubernetes service account that requires access.
-
-Role for Consul servers:
-```shell-session
-$ vault write auth/kubernetes/role/consul-server \
-    bound_service_account_names=<Consul server service account> \
-    bound_service_account_namespaces=<Consul installation namespace> \
-    policies=consul-server \
-    ttl=1h
-```
-
-To find out the service account name of the Consul server,
-you can run:
-
-```shell-session
- $ helm template --release-name ${RELEASE_NAME} --show-only templates/server-serviceaccount.yaml hashicorp/consul
-```
-
-Role for Consul clients:
-
-```shell-session
-$ vault write auth/kubernetes/role/consul-client \
-    bound_service_account_names=<Consul client service account>  \
-    bound_service_account_namespaces=default \
-    policies=ca-policy \
-    ttl=1h
-```
-
-To find out the service account name of the Consul client, use the command below.
-```shell-session
- $ helm template --release-name ${RELEASE_NAME} --show-only templates/client-serviceaccount.yaml hashicorp/consul
-```
-
-Role for CA components:
-```shell-session
-$ vault write auth/kubernetes/role/consul-ca \
-    bound_service_account_names="*" \
-    bound_service_account_namespaces=<Consul installation namespace> \
-    policies=ca-policy \
-    ttl=1h
-```
-
-The above Vault Roles will now be your Helm values for `global.secretsBackend.vault.consulServerRole` and
-`global.secretsBackend.vault.consulCARole` respectively.
-
-### Update the Consul on Kubernetes helm chart
+    <CodeBlockConfig filename="ca-policy.hcl">
+
+    ```HCL
+    path "pki/cert/ca" {
+      capabilities = ["read"]
+    }
+    ```
+
+    </CodeBlockConfig>
+
+    ```shell-session
+    $ vault policy write ca-policy ca-policy.hcl
+    ```
+
+    -> **Note:** The PKI secret path referenced by the above Policy will be your `global.tls.caCert.secretName` Helm value.
+
+1. Configure allowed domains for PKI certificates
```

Comment:
```suggestion
## Configure allowed domains for PKI certificates.
```
If I'm following along correctly, this is a new procedure and not part of configuring the policy, correct?